examples/sphere/shallow_water.jl

```diff
@@ -366,21 +351,13 @@ else # steady-state and mountain test cases share the same form of fields
 
         return (h = h, u = u)
     end
+    return Y
 end
-if !usempi
-    Y0_global = deepcopy(Y)
-else
-    Y0_global_values = DataLayouts.gather(context, Fields.field_values(Y))
-    if ClimaComms.iamroot(context)
-        Y0_global = Fields.Field(Y0_global_values, global_space)
-    end
-end
-ghost_buffer = usempi ? Spaces.create_ghost_buffer(Y) : nothing
 
 function rhs!(dYdt, y, parameters, t)
     @nvtx "rhs!" color = colorant"red" begin
-        f = parameters.f
-        h_s = parameters.h_s
+        (; f, h_s, ghost_buffer, params) = parameters
```

Comment:
What's the difference between this and the `unpack` macro ?

Reply:
Destructuring is now natively supported by Julia. We do not need to use an external package.